src/organizations/OrganizationEntity.js

```diff
@@ -35,6 +35,44 @@ export class OrganizationEntity extends Entity {
 		return this._entity && this._entity.properties && this._entity.properties.isActive;
 	}
 
+	processedDate(hideCourseStartDate, hideCourseEndDate) {
+		var nowDate = Date.now();
+		var startDate = Date.parse(this.startDate());
+		var endDate = Date.parse(this.endDate());
+		var dateType = null;
+		var date = null;
+
+		if (startDate > nowDate) {
+			dateType = 'startsAt';
+			date = new Date(startDate);
+			if (hideCourseStartDate) {
+				return null;
+			}
+		} else if (endDate < nowDate) {
+			dateType = 'ended';
+			date = new Date(endDate);
+			if (hideCourseEndDate) {
+				return null;
+			}
+		} else if (endDate >= nowDate) {
+			dateType = 'endsAt';
+			date = new Date(endDate);
+			if (hideCourseEndDate) {
+				return null;
+			}
+		} else {
+			return null;
+		}
+
+		const dateTime = {
+			type: dateType,
+			date: date,
+			beforeStartDate: startDate ? startDate > nowDate : null,
+			afterEndDate: endDate ? endDate <= nowDate : null
+		}
+		return dateTime;
+	}
+
```

Comment:
Copied logic from https://github.com/BrightspaceHypermediaComponents/organizations/blob/master/components/d2l-organization-date/d2l-organization-date.js#L76

Reply:
will remove it from Organization once it's merged with new release